main.go

```diff
@@ -21,7 +20,11 @@ func main() {
 
 	e.GET("/", api.RootHandler)
 
-	e.GET("/apps", api.AppsHandler)
+	e.GET("/apps", api.BuildAppsHandler(db))
+	e.GET("/users", api.BuildUsersHandler(db))
+	e.GET("/jobs/counts", api.BuildJobsHandler(db))
+	e.GET("/logins", api.BuildLoginCountHandler(db))
+	e.GET("/logins/distinct", api.BuildDistinctLoginCountHandler(db))
```

Comment:
Another option might be to have `/logins` return both the total number of logins and the number of distinct logins. The primary advantage of combining the service endpoints like that is that gathering the stats will require fewer service calls. The primary disadvantage would be that both stats would be obtained even in cases where the caller only wants one of them.

In this case, the overhead of the service call probably outweighs the overhead of gathering both stats, so it might be worthwhile to consolidate. I don't have an extremely strong opinion on this, though. I'll leave the choice up to you.

Reply:
I went with your suggestion and consolidated the endpoints.